apps/api/src/app/integrations/dto/create-integration.dto.ts

```diff
@@ -1,20 +1,15 @@
 import { IsDefined } from 'class-validator';
-import { ChannelTypeEnum } from '@notifire/shared';
+import { ChannelTypeEnum, ICreateIntegrationBodyDto, ICredentialsDto } from '@notifire/shared';
 
-export class CredentialsDto {
-  apiKey: string;
-
-  secretKey: string;
-}
-export class CreateIntegrationBodyDto {
+export class CreateIntegrationBodyDto implements ICreateIntegrationBodyDto {
```

Comment:
@djabarovgeorge Iove the shared with the client DTO. I wish we could share this entire class to avoid duplication. For now it's great but I would love to discuss the shared client-api DTO's and maybe we can find a better approach without duplication.

Reply:
I totally agree and would love to go over it